pkg/localkube/dns.go

```diff
@@ -237,11 +237,11 @@ func createEndpoint(client *kubeclient.Client, meta kube.ObjectMeta, dnsIP strin
 				Ports: []kube.EndpointPort{
 					{
 						Name: "dns",
-						Port: dnsPort,
+						Port: int32(dnsPort),
```

Comment:
Is this the int -> int32 thing you talked mentioned :smile:?
I think that we always should explicitely assign `int32` or `int64` to avoid 32/64-bit problems.
In this case we'll clean it up later